nginx.conf

```diff
@@ -0,0 +1,43 @@
+
+error_log  logs/error.log  debug;
+
+events {
+    worker_connections  1024;
+}
+
+
+http {
+    include       mime.types;
+    default_type  application/octet-stream;
+
+    sendfile        on;
+    #tcp_nopush     on;
+
+    keepalive_timeout  65;
+    #gzip  on;
+
+    # DataDome configuration
+    lua_ssl_verify_depth 3;
```

Comment:
Is this required or is it default value ?
if it's default it can be omitted

Reply:
It is required, it cannot be omitted.